packages/ember-data/lib/serializers/json_serializer.js

```diff
@@ -745,7 +745,7 @@ export default Ember.Object.extend({
     call is made to `DS.Store#findQuery`. By default this method is an
     alias for [extractArray](#method_extractArray).
 
-    @method extractFindQuery
+    @method extractFindQueryOne
```

Comment:
Good catch! I Guess I mixed names while changing from `findQuery` to `findQueryOne`. Thanks :)